src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -274,16 +275,10 @@ private void testSortAction() throws Exception {
                            "Danny Engrid",
                            "Emily");
 
-        verifySortingOrderAscending(By.id("button_sortToTeam"),
-                "Team 1{*}Team 2",
-                "Team 1{*}Team 2",
-                "Team 2{*}Team 2",
-                "Team 2{*}Team 3");
-
-        verifySortingOrderDescending(By.id("button_sortToTeam"),
-                "Team 1{*}Team 2",
-                "Team 1{*}Team 2",
-                "Team 1{*}Team 2",
+        verifySortingOrder(By.id("button_sortToTeam"),
```

Comment:
Add a comment on how the sorting doesn't actually test properly? Not to mention I think these tests on sorting can be difficult to understand.